### ISSUE TITLE
About The Performance

### ISSUE THREAD

**Issue by HankLiu10** [archon159/RAFT#1]
Dear Authors, I trained your models on ETTh1. The results show even better performance than the paper. I tried your code when I saw your paper, so I'd like to know if it's because I wrongly set the params.

![Image](https://github.com/user-attachments/assets/2dda832d-0609-49aa-9ea5-97fc6f5dbc00)

**Comment by Sungwon-Han** [comment]
Thank you for your interest in our paper.
The reported performance metrics (e.g., Table 1) are averaged over multiple prediction lengths - 96, 192, 336, and 720 - and thus may show higher MSE compared to results at a single prediction length like 96.
As you pointed out, various hyper-parameters such as input shape can significantly impact performance. 
Please note that we performed hyper-parameter tuning on the validation set before reporting the final results (see implementation details section).

**Comment by HankLiu10** [comment]
Thanks for your quick reply. Congratulations on your excellent work!
